collectors/proc.plugin/multi_metadata.yaml

```diff
@@ -174,20 +172,22 @@ modules:
       plugin_name: proc.plugin
       module_name: /proc/sys/kernel/random/entropy_avail
       monitored_instance:
-        name: proc /proc/sys/kernel/random/entropy_avail
+        name: Entropy
         link: ''
-        categories: []
-        icon_filename: ''
+        categories:
+         - data-collection.linux-systems.system-metrics
+        icon_filename: 'syslog.png'
       related_resources:
         integrations:
           list: []
       info_provided_to_referring_integrations:
         description: ''
-      keywords: []
+      keywords:
+       - entropy
       most_popular: false
     overview:
       data_collection:
-        metrics_description: ''
+        metrics_description: 'Examine Entropy metrics for insights into random number generation.'
```

Comment:
I would suggest leaving this empty. There’s a single metric, and there will _always_ be a single metric, so not much point in providing a description at the top.

Reply:
will leave empty